packages/request-node/src/requestNode.ts

```diff
@@ -162,93 +174,46 @@ class RequestNode {
     this.express.use(express.json());
     this.express.use(express.urlencoded({ extended: true }));
 
-    // Route for health check
-    router.get('/healthz', (_, serverResponse: any) => {
-      return serverResponse.status(httpStatus.OK).send('OK');
-    });
-
-    // Route for readiness check
-    router.get('/readyz', (_, serverResponse: any) => {
-      if (this.initialized) {
-        return serverResponse.status(httpStatus.OK).send('OK');
-      }
-      return serverResponse.status(httpStatus.SERVICE_UNAVAILABLE).send(NOT_INITIALIZED_MESSAGE);
-    });
-
-    // Route for status check
-    router.get('/status', (clientRequest: any, serverResponse: any) => {
-      if (this.initialized) {
-        return getStatus(clientRequest, serverResponse, this.dataAccess, this.logger);
-      } else {
-        return serverResponse.status(httpStatus.SERVICE_UNAVAILABLE).send(NOT_INITIALIZED_MESSAGE);
-      }
-    });
-
-    // Route for ipfs-add request
-    router.post('/ipfsAdd', (clientRequest: any, serverResponse: any) => {
-      if (this.initialized) {
-        return ipfsAdd(clientRequest, serverResponse, this.ethereumStorage, this.logger);
-      } else {
-        return serverResponse.status(httpStatus.SERVICE_UNAVAILABLE).send(NOT_INITIALIZED_MESSAGE);
-      }
-    });
-
-    // Route for persistTransaction request
-    router.post('/persistTransaction', (clientRequest: any, serverResponse: any) => {
-      if (this.initialized) {
-        return this.persistTransaction.persistTransaction(
-          clientRequest,
-          serverResponse,
-          this.dataAccess,
-          this.logger,
-        );
-      } else {
-        return serverResponse.status(httpStatus.SERVICE_UNAVAILABLE).send(NOT_INITIALIZED_MESSAGE);
-      }
-    });
+    router.get('/healthz', (_, res) => res.status(StatusCodes.OK).send('OK'));
+    router.use(this.initializedMiddelware());
+    router.get('/readyz', (_, res) => res.status(StatusCodes.OK).send('OK'));
+    router.get('/status', this.getStatusHandler.handler);
+    router.post('/ipfsAdd', this.ipfsAddHandler.handler);
+    router.post('/persistTransaction', this.persistTransaction.handler);
+    router.get('/getConfirmedTransaction', this.confirmedTransactionStore.getConfirmedTransaction);
```

Comment:
Just curiousity, why there is no `handler` here?

Reply:
I hesitated, actually all other handlers are based on this one 
but this class is a "store" for a given type of data

I was thinking to later introduce a Handler for this endpoint, separated from the Store

Reply:
actually I did this change ;)